app/controllers/admin/searchTrackHoursVolunteer.py

```diff
@@ -0,0 +1,40 @@
+from flask import request, render_template, redirect, url_for, request, flash
+from flask import json, jsonify
+from app.controllers.admin import admin_bp
+from app.models.event import Event
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+from app.models.user import User
+from peewee import *
+
+@admin_bp.route('/searchTrackHoursVolunteers/<query>', methods = ['GET'])
+def searchTrackHoursVolunteers(query):

```

Comment:
This search functionality needs to be in a separate function so you can test it without having to go through the ui